### PR TITLE
Novitsky/change about portal implementation

### PR DESCRIPTION
**Pull request by SergeyNovitsky**

### CODE REVIEW

**Review by VadymLevkovskyi — APPROVED** [review]

**Review by DmyMi — APPROVED** [review]

File: OutOfSchool/OutOfSchool.DataAccess/Repository/AboutPortalRepository.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        private readonly OutOfSchoolDbContext db;
+
+        public AboutPortalRepository(OutOfSchoolDbContext dbContext)
+            : base(dbContext)
+        {
+            db = dbContext;
+        }
+
+        public IUnitOfWork UnitOfWork => db;
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.DataAccess/Repository/AboutPortalRepository.cs:19]: Don't see if `UnitOfWork` is used anywhere. Probably remove it.

File: OutOfSchool/OutOfSchool.DataAccess/Repository/IAboutPortalRepository.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.Services.Repository
+{
+    public interface IAboutPortalRepository : ISensitiveEntityRepository<AboutPortal>
+    {
+        IUnitOfWork UnitOfWork { get; }
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.DataAccess/Repository/IAboutPortalRepository.cs:11]: Don't see if `UnitOfWork` is used anywhere. Probably remove it.

**Review by VadymLevkovskyi — CHANGES_REQUESTED** [review]
Minor changes requested, see comments

File: OutOfSchool/OutOfSchool.DataAccess/Repository/AboutPortalRepository.cs (hunk 1 above)
Comment (SergeyNovitsky) [reply OutOfSchool/OutOfSchool.DataAccess/Repository/AboutPortalRepository.cs]: I removed it.

File: OutOfSchool/OutOfSchool.DataAccess/Repository/IAboutPortalRepository.cs (hunk 2 above)
Comment (SergeyNovitsky) [reply OutOfSchool/OutOfSchool.DataAccess/Repository/IAboutPortalRepository.cs]: Removed it.

**Pull request merged**